Sources/web3swift/Web3/Web3+Contract.swift

```diff
@@ -16,7 +16,7 @@ extension web3 {
 
     /// Web3 instance bound contract instance.
     public class web3contract {
-        var contract: EthereumContract
+        private(set) public var contract: EthereumContract
```

Comment:
This is another turning point.
I'm not sure of the reasons why `var contract: EthereumContract` wasn't initially public. It holds data that could be useful for developers and that is first of all decoded and filtered ABI.

Reply:
I've looked at whole `web3contract` implementation why is it even variable? As i've seen in code there's none assigning to its happening except of init. Should we made it constant?

Reply:
Good point. I'll take a look at this and other variables in `web3contract` and will propose changes making them constant if no new question will pop up during this additional review.
I wasn't sure if it should stay as `var` but also wanted to make it `let`. It's good that you suggested making it a constant 🙂